src/configs/axios.ts

```diff
@@ -3,13 +3,10 @@ import axios from 'axios';
 import { PAGE_URL } from './path';
 
 export const API = axios.create({
-  baseURL: import.meta.env.VITE_DEV_SERVER_URL,
-
-  // baseURL: 'https://causw-dev.du.r.appspot.com',
-
-  //process.env.REACT_APP_ENV === 'production'
-  //   ? 'https://causw-prod.du.r.appspot.com'
-  //   : 'https://causw-dev.du.r.appspot.com',
+  baseURL:
+    process.env.NODE_ENV !== 'development'
```

Comment:
사소하긴 한데 삼항연산자로 쓸 땐 `process.env.NODE_ENV === 'production'` 으로 작성하는 게 머리에 더 잘들어오는 것 같아요 ㅎㅎ
근데 로직에는 문제가 없습니다!

Reply:
오 찾아보니까 vite에선 `import.meta.env.PROD` 이랑 `import.meta.env.DEV` 을 따로 제공해주네요!

아래 쓴 거처럼 바꿔도 괜찮을 것 같아요
```suggestion
    import.meta.env.PROD ? import.meta.env.VITE_PROD_SERVER_URL : import.meta.env.VITE_DEV_SERVER_URL
```